WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Reprint Shipping Label/ReprintShippingLabelCoordinator.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+import Yosemite
+
+/// Coordinates navigation actions for reprinting a shipping label.
+final class ReprintShippingLabelCoordinator {
```

Comment:
Idea: I think it would be nice to mention in the comments to the `ReprintShippingLabelCoordinator` that we're intentionally retaining `self` in callbacks to avoid confusion.

Reply:
Makes sense! added to `showReprintUI` in https://github.com/woocommerce/woocommerce-ios/pull/3374/commits/267dda34b2cdf7a122618e2ec90b1dffb6a6f9a7